src/renderer/components/NewProjectModal/ModalContainer.tsx

```diff
@@ -16,7 +16,10 @@ const CustomModal = styled(Modal)`
 
 const CustomModalInner = styled(Box)`
   background: ${colors.grey[700]};
-  padding: 30px;
+  padding-left: 30px;
```

Comment:
This can be simplified to one line. :) .Here is an example:
![image](https://user-images.githubusercontent.com/101784287/167562159-e4ab3a2c-c406-47f6-8952-97e1d43d634f.png)